compiler/natives/src/crypto/rand/rand.go

```diff
@@ -5,6 +5,7 @@ package rand
 import (
 	"errors"
 
+	"github.com/gopherjs/gopherjs/compiler/natives/src/crypto/webcrypto"
```

Comment:
I think `github.com/gopherjs/gopherjs/js/webcrypto` might be better, since `natives` is not meant to be imported directly.